### ISSUE TITLE
Functions do not work with column N:N

### ISSUE THREAD

**Issue by u2618** [audreyt/ethercalc#534]
Hi, 

when I try to sum up all values of column N, it does not work. Other columns work like expected.

` =SUM(N:N) `results in „Falsche Argumente für Funktion N“ (wrong arguments for function N)

N seems to be a fuction. Other Columns that do not work are T, FV, etc.

I tested with Chromium and Opera browsers.

Thanks,

**Comment by eddyparkinson** [comment]
Agree, this is a bug. Both N() and T() are functions and looks like it is getting them mixed up.

I would suspect this function
https://github.com/marcelklehr/socialcalc/blob/master/formula1.js#L152
but did not double check.
